samples/cellular/nrf_cloud_rest_cell_location/src/main.c

```diff
@@ -367,52 +360,10 @@ static void do_jitp(void)
 }
 #endif
 
-#define REQ_CARD "---> Press button %d to enable the location card for this device on nrfcloud.com"
-static void request_location_card_enable(void)
```

Comment:
@tony-le-24 does this removal affect your sample tests?

Reply:
yes, he already made a PR to fix.